recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -23,7 +23,7 @@ conda.exe config --set channel_priority %channel_priority%
 
 :: Set the conda-build working directory to a smaller path
 if "%CONDA_BLD_PATH%" == "" (
-    set "CONDA_BLD_PATH=C:\\bld\\"
+    set "CONDA_BLD_PATH=C:\bld"
```

Comment:
There are feedstocks (like [`openvino`]( https://github.com/conda-forge/openvino-feedstock/blob/290a9ef7987fba531f0b342443b5a7075621b276/.azure-pipelines/azure-pipelines-win.yml#L33 )), which already do this today. So this should work more generally \